packages/ui/src/lib/Markdown/index.tsx

```diff
@@ -1,4 +1,5 @@
-import * as React from 'react'
+'use client'
```

Comment:
this one is on an index file because it exports an object and I think you will otherwise run into the "cannot dot into a client component" problem